### ISSUE TITLE
Certificate of Incorporation should be "Amended and Restated"

### ISSUE THREAD

**Issue by goodcounsel** [seriesseed/equity#28]
The Series Seed Certificate is styles as a "Restated" Certificate but it is not just Restated. It is Amended and Restated. This should be reflected throughout the Certificate.

**Comment by kemitchell** [comment]
@goodcounsel: This begs an interesting question. Is there any common law legal significance to "amended [and restated]"? The phrase is definitely conventional among attorneys I know.

The Delaware code actually just uses "restated":

> § 245 Restated certificate of incorporation.
> 
> (a) A corporation may, whenever desired, integrate into a single instrument all of the provisions of its certificate of incorporation which are then in effect and operative as a result of there having theretofore been filed with the Secretary of State 1 or more certificates or other instruments pursuant to any of the sections referred to in § 104 of this title, and it may at the same time also further amend its certificate of incorporation by adopting a restated certificate of incorporation.

http://delcode.delaware.gov/title8/c001/sc08/index.shtml

**Comment by goodcounsel** [comment]
Yes, there is a clear difference.

Restating is the act of taking separate amendments that have already been filed and incorporating them into a single document. (Sometimes amendments are just filed as the individual item, without incorporating into the body of the original.)

Amending is the making of the changes. That is covered in Section 242 of the Delaware Code.

**Comment by rickcolosimo** [comment]
Yes, there's a difference. Yes, we should follow convention (making a note to myself to do a pull request on this). I have to re-read the form.

No, I don't think anyone's determined that there's an authoritative legal significance to the title, notwithstanding that if it says "amended" you might expect other effective provisions in another document and that if it says "restated" you might expect that it is only restating pre-existing amendments and not making new ones.

Thanks,

Rick

Richard J. Colosimo
rick@thoughtstorm.com
m 646-701-1080

**Comment by kemitchell** [comment]
@goodcounsel, @rickcolosimo: What struck me is that 245 speaks of what I'd call A&R certificates as just "restated certificates". A bit further on in the stat:

> § 245 Restated certificate of incorporation.
> 
> (b) [...] _If the restated certificate of incorporation restates and integrates and also further amends in any respect the certificate of incorporation_, as theretofore amended or supplemented, it shall be proposed by the directors and adopted by the stockholders in the manner and by the vote prescribed by § 242 of this title or [...] [Italics added]

Hence my question about common law, on the off chance there's some notice (rather than style or good housekeeping) value in "amended" at the top of a filing.

I am all for changing the form, in deference to the Principle of Least Surprise. But I am constantly surprised how many stylistics orthodoxies in Delaware corporate law [aren't firmly grounded in the code](http://www.theconglomerate.org/2006/05/shareholder_v_s.html).

**Comment by thought-criminal** [comment]
Testing...

**Comment by jrmiller82** [comment]
If you make changes but call it 'restated' only to investors, you may be
violating federal and state securities regulations (fraud /
misrepresentation) as 'restated' implies that it's a combination of
existing filings into one cohesive document as opposed to making new
changes. Hence 'amended and restated'. I doubt you'd ever find any state's
code to be that explicit.

**Comment by rickcolosimo** [comment]
James: Above, Kyle (https://github.com/seriesseed/equity/issues/28#issuecomment-98172891 and https://github.com/seriesseed/equity/issues/28#issuecomment-98180309) twice posted the exact language from the relevant Delaware statute. Hence the position that while the statute is crystal clear that "restated" includes what lawyers commonly refer to as "amended and restated." So the code is exactly that specific.

You'll see his notes and mine indicating that we're in favor of adopting conventional usage even though the plain language of the Delaware code supports just using "restated." Going back to Kyle's question

> common law legal significance

I think that the answer is no - I've never read or seen a case that says that calling the document one thing or another in this way, beyond the specific requirement in the code that in fact implies that this "amended and restated" approach might violate the law:

> (c) A restated certificate of incorporation shall be specifically designated as such in its heading. 
([http://delcode.delaware.gov/title8/c001/sc08/index.shtml](http://delcode.delaware.gov/title8/c001/sc08/index.shtml))

But I'm comfortable believing that as long as the words "restated certificate of incorporation" appear in the heading, the words "Amended and" don't violate the statute.

**Comment by jrmiller82** [comment]
The Delaware statute probably would not matter much for a non disclosure / misrepresentation securities lawsuit. What matters is whether it was misleading in your prospectus / PPM from the investors' perspective.

**Comment by goodcounsel** [comment]
This is an ancient thread; I don't know if this will even get to the right
place. Or if anyone is even maintaining these documents any more. But I am
just writing again because some lawyers in my firm noted the point about
the SeriesSeed Certificate of Incorporation using only "restated," and I
thought to myself, gee, that issue sounds familiar.

It seems as if the consensus was to make the change (per thread) but it
appears the change was never made.

Not a big deal, but I am curious.

---

Kenneth Obel
